pages/about.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8" />
+	<title>cathos/about</title>
+	<!-- Link to Style Sheet -->
+	<link href="styles/style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Eczar&display=swap" rel="stylesheet"> 
 </head>
+
 <body>
-  
+    <header>
+        <a id="site_title" href="index.html">
+            <h1>cathos</h1>
+        </a>
+        <nav>
+            <a href="about.html">about</a>
+            <a href="portfolio.html">portfolio</a>
+            <a href="contact.html">contact</a>
+            <a href="https://github.com/cathos">code</a>
+        </nav>
+    </header>
+    <section>
+        <h2>about</h2>
+        <p>Hi! I'm cathos, though you may know me by other names in real life.</p>
+        <p>I'm curious about lots of things, and I've worked in many fields and pursued a lot of hobbies.</p>
+        <p>I'm currently studying software development at <a href="https://adadevelopersacademy.org/">Ada Developer's Academy</a>.</p>
+        <p>I recently completed a certificate in Soul Care from the Soul Care Institute, and I'm interested in spirituality and mysticism, primarily from a Christian background.</p>
+        <p>I'm passionate about coffee and tea. I was a coffee roaster at Breckenridge Coffee Roasters for four years. I currently have a 1-kilo roaster, and I roast for friends and family.</p>
+        <p>I've worked off-and-on for years as an architectural drafter.</p>
+        <p>I studied Soil & Water Science in college, but did not finish a degree.</p>
+        <p>Decentralized, small, and low-tech technology are a big focus of my thoughts right now.</p>
+        <p>I'm passionate about social justice and ecological justice.</p>
+        <p>I love to tinker, cook, and make things!</p>
```

Comment:
I see that you chose to do multiple p tags here. You could use one large p tag and put a br tag when you want to create a break or new line in your text. You could also create an unordered list and list out these statements. You could also use the pre tag which preserves both spaces and line breaks. More info here https://www.w3schools.com/html/tryit.asp?filename=tryhtml_pre